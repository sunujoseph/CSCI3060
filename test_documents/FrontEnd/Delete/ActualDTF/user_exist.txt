02 nonusername AA 000000000
