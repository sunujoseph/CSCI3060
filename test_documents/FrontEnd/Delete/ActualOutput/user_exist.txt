path recieved: C:\Users\pwner\Documents\Uoit\Year 5 Semester 2\CSCI 3060U\Project\CSCI3060\test_documents\FrontEnd\Delete\ActualDTF\user_exist.txt
Enter Username: temp: |
delete
noÍÍÍÍÍÍÍııııİİİJeœ'İ|
username: |admin1
delete
n| not found
Error: Not Logged In
Error: Not Logged In
