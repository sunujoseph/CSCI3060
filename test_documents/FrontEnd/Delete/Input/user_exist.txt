login
admin1
delete
nonusername
