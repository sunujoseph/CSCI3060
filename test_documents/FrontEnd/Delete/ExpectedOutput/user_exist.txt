Enter user name: 
Enter user name to be deleted: 
Cannot delete, user does not exist.
